lib/isCI.js

```diff
@@ -0,0 +1 @@
+module.exports = Boolean(process.env.CI || process.env.BUILDKITE_BUILD_ID);
```

Comment:
Perhaps I'm misunderstanding this but buildkite already sets a `CI` env var (https://buildkite.com/docs/pipelines/environment-variables). Also, if you were trying to detect buildkite specifically, would the `BUILDKITE` env var be more appropriate here?

Reply:
The issue is that the Buildkite docker plugin doesn't forward any of those vars by default and most teams don't set it up. However, `BUILDKITE_BUILD_ID ` is forwarded by default.